### DISCUSSION TITLE
Override hostname?

### DISCUSSION THREAD

**Discussion by gnomemade** [ohmybash/oh-my-bash#317] [Q&A]
I've always set a static hostname in my PS1 because I'm not fond of my company's naming scheme. I'm a very new user of oh-my-bash, and I can't figure out how to override this. I see where SHORT_HOST is defined oh-my-bash.sh, and I tried messing with this var in ~/.bashrc, but I still get the "real" hostname from `scutil --get ComputerName` Any pointers on how I can override this for all themes?

**Comment by akinomyoga** [comment]
> I'm a very new user of oh-my-bash, and I can't figure out how to override this.

There is no mechanism for overriding the hostname. You can manually modify the theme files following [the instruction in README](https://github.com/ohmybash/oh-my-bash#customization-of--plugins-and-themes) (i.e., first copy the theme file to the custom directory and edit the copy). In particular, you can replace `\h` with the string you like.
